.github/workflows/CI_on_push.yml

```diff
@@ -0,0 +1,41 @@
+
+name: CI on push to main or issue-2 branch
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - issue-2-Setup_CI/CD_for_reference-models
```

Comment:
Let's remove this before merging this branch.

Reply:
Done!